### PR TITLE
Metal festival/Project week 3/Buisness-site

### PR DESCRIPTION
**Pull request by bloeli**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -3,28 +3,189 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Metal Festival &#x1F918;</title>
+    <link rel="stylesheet" href="./style.css">
```
Comment (bealun) [inline code/index.html:7]: I really wanna see a favicon 🥺 If not this week then next, right?

File: code/index.html [hunk 2]
```diff
@@ -3,28 +3,189 @@
...
+    <!-- SIGN UP FORM -->
+      <form 
+        class="sign-up-form" 
+        action="https://httpbin.org/anything" 
+        method="POST"
+      >
+        <h2>SIGN UP</h2>
+        <label for="firstName">First name:</label>
+        <br>
+        <br>
```
Comment (bealun) [inline code/index.html:57]: Using `<br>`is considered bad practise - try to use CSS styling to make it look the way you'd like instead!

File: code/index.html [hunk 3]
```diff
@@ -3,28 +3,189 @@
...
+          <h3 id="accordion-header-1" class="accordion-header">
+              <span>Food and drinks?</span>
+              <span>▼</span>
+          </h3>
+          <div class="accordion-text-container">
+              <p class="accordion-text">
+                We offer a range of food and beverage stalls inside the festival area as well as in the camping area. There are Vegan alternatives at some of the stalls. Water is free at the festival area and the camping area.
+              </p>
+          </div>
+        </div>
```
Comment (bealun) [inline code/index.html:133]: Well done with the code structure, this will pay off! 👏

**Review by bealun — APPROVED** [review]
What a nice projects you've made, really well done 🌟 I'm very impressed by your commit messages and how informative they are - which is a really good practise within tech. You code is easy to read and well structured so keep pushing forward. I do want to see some validation in your form but I'm not gonna request any changes for now - this is more for your own good.
Be proud of yourself and the work you've done this week, it's impressive! 👏

File: code/index.html (hunk 1 above)
Comment (bloeli) [reply code/index.html:7]: Ah thanks, have'nt thought/learned about it, but now I and my teammate in project week 4 added a favicon :)

File: code/index.html (hunk 2 above)
Comment (bloeli) [reply code/index.html:57]: Ah okay :)

File: code/index.html [hunk 4]
```diff
@@ -3,28 +3,189 @@
...
+        method="POST"
+      >
+        <h2>SIGN UP</h2>
+        <label for="firstName">First name:</label>
+        <br>
+        <br>
+          <input 
+            type="text" 
+            name="firstNameValue" 
+            id="firstName"
```
[the remark this answers is not in the document]
Comment (bloeli) [reply code/index.html:61]: I see, thanks :)

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,310 @@
...
+  }
+  
+  /* ACCORDION */
+  .accordion {
+    width: 70%;
+  } 
+}
+
+/* MEDIA FOR TABLET */
+@media (min-width: 668px) and (max-width: 1024px){
```
[the remark this answers is not in the document]
Comment (bloeli) [reply code/style.css:259]: Thank you! :D

**Comment by bloeli** [comment]
> What a nice projects you've made, really well done 🌟 I'm very impressed by your commit messages and how informative they are - which is a really good practise within tech. You code is easy to read and well structured so keep pushing forward. I do want to see some validation in your form but I'm not gonna request any changes for now - this is more for your own good.
> Be proud of yourself and the work you've done this week, it's impressive! 👏

**Pull request closed without merging**